csv-data-management/requirements.txt

```diff
@@ -1,3 +1,5 @@
 # Add Python Dependencies
 pandas
 numpy
+polars
+pyarrow
```

Comment:
where do you use this one? I don't see related import statements

Reply:
> where do you use this one? I don't see related import statements
Pyarrow is being internally used by Polars for some functionalities.

Reply:
> where do you use this one? I don't see related import statements
I was using different approach at the time of importing this, But now it's not necessary. I have removed this.